server/kit/kitserver.go

```diff
@@ -91,34 +91,38 @@ func NewServer(svc Service) *Server {
 
 	projectID := googleProjectID()
 	if projectID != "" {
-		svcName, svcVersion := "default", "v1.0"
+		svcName, svcVersion := "", ""
 		if isGAE() {
 			_, svcName, svcVersion = getGAEInfo()
 		}
 		if n, v := os.Getenv("SERVICE_NAME"), os.Getenv("SERVICE_VERSION"); n != "" {
 			svcName, svcVersion = n, v
 		}
 
-		err = initSDExporter(sdExporterOptions(projectID, svcName, svcVersion, lg))
-		if err != nil {
-			lg.Log("error", err,
-				"message", "unable to initiate error tracing exporter")
-		}
-
-		propr = &sdpropagation.HTTPFormat{}
-
-		errs, err = errorreporting.NewClient(ctx, projectID, errorreporting.Config{
-			ServiceName:    svcName,
-			ServiceVersion: svcVersion,
-
-			OnError: func(err error) {
+		if opt := sdExporterOptions(projectID, svcName, svcVersion, lg); opt != nil {
+			err = initSDExporter(*opt)
+			if err != nil {
 				lg.Log("error", err,
-					"message", "error reporting client encountered an error")
-			},
-		})
-		if err != nil {
-			lg.Log("error", err,
-				"message", "unable to initiate error reporting client")
+					"message", "unable to initiate error tracing exporter")
+			}
+
+			propr = &sdpropagation.HTTPFormat{}
+
+			if err == nil {
```

Comment:
I don't think the exporters should prevent the error reporting from being initiated. If anything, that may be when we need the error reporting the most?

Let's remove this `if` statement and just let this fail on its own, if there are environment issue.